### ISSUE TITLE
Popup not working in django1.9

### ISSUE THREAD

**Issue by Dharmik8478** [django-admin-bootstrapped/django-admin-bootstrapped#232]
In foreign key field the edit button is not working and + button opens it in the next page (not to the popup).

**Comment by xrmx** [comment]
Yeah, django 1.9 is possibly broken because templates have changed. And possibly it was me breaking popups for a good cause though :) https://github.com/django/django/commit/0decef337f7f00d73b380531ac22cbc843d6e806

**Comment by Dharmik8478** [comment]
Thanks for quick reply, it works for me...
